src/classes/services/content_extractor.py

```
@@ -0,0 +1,232 @@
+            name = meta.get('name') or meta.get('property')
+            content = meta.get('content')
+            
+            if name and content:
+                metadata[name] = content
+        
+        # Extract JSON-LD structured data
+        for script in soup.find_all('script', type='application/ld+json'):
+            try:
+                data = json.loads(script.string)
+                metadata['json_ld'] = data
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)